index.html

```diff
@@ -51,33 +58,185 @@ <h1 class="about-title padding">I'm Rose Glad to see you!</h1>
     </section>
 
     <section class="middle-section">
-      <div class="card project1">
+      <div class="card project1" >
         <div class="project-image">
           <img src="./images/SnapshootPortfolio.svg" alt="" srcset="">
         </div>
         <h3 class="project-title">Tonic</h3>
         <ul class="project-links">
           <li class="canopy">CANOPY</li>
-          <div class="small-circle">.</div>
+
+          <li class="small-circle">.</li>
+          <li class="career-role">Back End Dev</li>
+          <li class="small-circle">.</li>
           <li class="role">Back End Dev</li>
+
+          <li class="year">2015</li>
+        </ul>
+        <div class="project-description">
+          <p>A daily selection of privately personalized
+            reads; no accounts or sign-ups required.</p>
+        </div>
+
+        <ul class="my-skills">
+          <li><a href="#" class="skill-link html-btn">html</a></li>
+          <li><a href="#" class="skill-link css-btn">css</a></li>
+          <li><a href="#" class="skill-link js-btn">javascript</a></li>
+        </ul>
+      
+        <a href="#" class="see-project-btn">See Project</a>
+      </div>
```

Comment:
Currently, your styles are not exactly lookalike to the Figme design template you have chosen. The difference mainly includes:-

1- Font-sizes
2- Margins and Paddings
2- List item's background color.

Please consider making it an exact copy of the [Figma Template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=37%3A2238&t=UPJfUKzuGg7shdH7-0) to make it look better and professional. 

## Your Design:-

![image](https://user-images.githubusercontent.com/94223367/219565945-cc96a8cc-51de-4d77-b18d-910e261ed751.png)